components/observations/ObservationDetailView.tsx

```diff
@@ -20,42 +22,74 @@ import {
   AvalancheDateUncertainty,
   AvalancheTrigger,
   AvalancheType,
+  CloudCover,
   FormatActivity,
   FormatAvalancheAspect,
   FormatAvalancheBedSurface,
   FormatAvalancheCause,
   FormatAvalancheDateUncertainty,
-  FormatAvalancheProblemDistribution,
   FormatAvalancheTrigger,
   FormatAvalancheType,
   FormatCloudCover,
+  FormatInstabilityDistribution,
   FormatPartnerType,
   FormatSnowAvailableForTransport,
   FormatWindLoading,
+  InstabilityDistribution,
   MapLayer,
-  PartnerType,
+  Observation,
+  observationSchema,
+  SnowAvailableForTransport,
+  WindLoading,
 } from 'types/nationalAvalancheCenter';
 import {utcDateToLocalTimeString} from 'utils/date';
 
+export const NWACObservationDetailView: React.FunctionComponent<{
+  id: string;
+}> = ({id}) => {
+  const observationResult = useNWACObservation(parseInt(id));
+  const observation = observationResult.data;
+  const mapResult = useMapLayer(observation?.center_id);
+  const mapLayer = mapResult.data;
+
+  if (incompleteQueryState(observationResult, mapResult)) {
+    return <QueryState results={[observationResult, mapResult]} />;
+  }
+
+  return <ObservationCard observation={observation} mapLayer={mapLayer} />;
+};
+
 export const ObservationDetailView: React.FunctionComponent<{
   id: string;
 }> = ({id}) => {
   const observationResult = useObservationQuery({
     id: id,
   });
   const observation = observationResult.data;
-  const mapResult = useMapLayer(observation?.getSingleObservation.centerId?.toUpperCase() as AvalancheCenterID);
+  const mapResult = useMapLayer(observation?.getSingleObservation.center_id?.toUpperCase() as AvalancheCenterID);
   const mapLayer = mapResult.data;
 
   if (incompleteQueryState(observationResult, mapResult)) {
     return <QueryState results={[observationResult, mapResult]} />;
   }
 
-  return <ObservationCard observation={observation.getSingleObservation} mapLayer={mapLayer} />;
+  const parseResult = observationSchema.deepPartial().safeParse(observation.getSingleObservation);
```

Comment:
@floatplane not sure if you saw `.deepPartial()` - causing all the possible `undefined`, but necessary because of how these APIs are ...